handbook/engineering/search/goals.md

```diff
@@ -2,64 +2,52 @@
 
 ## Goals
 
-Goals help us achieve our [vision](index.md#vision) for search.
+### Support large scale organizations
```

Comment:
I would like us to come up with a better title for the goal, ideally so it is more time based (has a clear "ending"). Also, this almost makes it seem like we don't support large scale _at all_, which is not true. We want this to capture that we are targeting an excellent experience and raising the bar for large organizations. I can't think of a better suggestion at the moment, would love some ideas.